src/instruments/src/EFB/StatusBar/QuickControls.tsx

```diff
@@ -0,0 +1,298 @@
+    const [simBridgeEnabled, setSimbridgeEnabled] = usePersistentProperty('CONFIG_SIMBRIDGE_ENABLED', 'AUTO ON');
+
+    const [showQuickControlsPane, setShowQuickControlsPane] = useState(false);
+    const [simBridgeConnected, setSimBridgeConnected] = useState(false);
+
+    // To prevent keyboard input (esp. END key for external view) to change
+    // the slider position. This is accomplished by a
+    // onAfterChange={() => sliderRef.current.blur()}
+    // in the Slider component props.
+    const brightnessSliderRef = useRef<any>(null);
+
+    const handleAutoBrightness = () => {
+        setUsingAutobrightness(usingAutobrightness ? 0 : 1);
+    };
+
+    const handleSleep = () => {
+        history.push('/');
+        power.setPowerState(PowerStates.STANDBY);
+    };
+
+    const handlePower = () => {
+        history.push('/');
+        power.setPowerState(PowerStates.SHUTOFF);
+    };
+
+    const handleAlignADIRS = () => {
+        const previousAlignTimeVar = adirsAlignTimeSimVar;
+        setAdirsAlignTimeSimVar(1);
+        setTimeout(() => {
+            setAdirsAlignTimeSimVar(previousAlignTimeVar);
+        }, 500);
+    };
+
+    const handleInstantBoarding = () => {
+        const previousBoardingRate = boardingRate;
+        setBoardingRate('INSTANT');
+        setTimeout(() => {
+            setBoardingRate(previousBoardingRate);
+        }, 500);
+    };
+
+    const handleResetSimBridgeConnection = () => {
+        if (simBridgeConnected) {
+            setSimbridgeEnabled('PERM OFF');
+            return;
+        }
+        setSimbridgeEnabled('AUTO ON');
+    };
+
+    const determineSimBridgeState = (): SimBridgeState => {
+        if (simBridgeConnected) return SimBridgeState.CONNECTED;
+        if (!simBridgeConnected && simBridgeEnabled === 'AUTO OFF') return SimBridgeState.OFFLINE;
+        if (!simBridgeConnected && simBridgeEnabled === 'AUTO ON') return SimBridgeState.CONNECTING;
+        return SimBridgeState.OFF;
+    };
+
+    const simBridgeButtonStyle = ():string => {
+        switch (determineSimBridgeState()) {
+        case SimBridgeState.CONNECTED: return 'bg-utility-green text-theme-body';
+        case SimBridgeState.CONNECTING: return 'bg-utility-amber text-theme-body';
+        case SimBridgeState.OFFLINE: return 'bg-utility-red text-theme-body';
+        default: return '';
+        }
+    };
+
+    const simBridgeButtonStateString = ():string => {
+        switch (determineSimBridgeState()) {
+        case SimBridgeState.CONNECTED: return t('QuickControls.SimBridgeConnected');
+        case SimBridgeState.CONNECTING: return t('QuickControls.SimBridgeConnecting');
+        case SimBridgeState.OFFLINE: return t('QuickControls.SimBridgeOffline');
+        default: return t('QuickControls.SimBridgeOff');
+        }
+    };
+
+    const handleSettings = () => {
+        history.push('/settings/flypad');
+    };
```

Comment:
These functions could be memoized with React.memo(), especially ones that are often used

Reply:
> These functions could be memoized with React.memo(), especially ones that are often used

Great tip!
I have done the last three function with use memo now - the others are action handlers so I do not think it makes sense for them.